docs/doxygen/doxygen.cfg

```diff
@@ -745,15 +745,18 @@ WARN_LOGFILE           = p4c-doxygen-warn.log
 
 # include first the Markdown documentation in docs and then the code
 INPUT                  = $(SRCDIR)/docs/doxygen \
+                         $(SRCDIR)/LICENSE.md   \
                          $(SRCDIR)/docs/README.md \
                          $(SRCDIR)/docs/IR.md   \
                          $(SRCDIR)/ir           \
+                         ir/ir-generated.h      \
                          $(SRCDIR)/frontends    \
                          $(SRCDIR)/midend       \
                          $(SRCDIR)/backends     \
                          $(SRCDIR)/lib          \
                          $(SRCDIR)/extensions   \
-                         ir/ir-generated.h
+                         $(SRCDIR)/control-plane/p4RuntimeSerializer.h \
+                         $(SRCDIR)/control-plane/p4RuntimeSerializer.cpp
```

Comment:
I'm guessing you're specifying these files explicitly to avoid the PI submodule? Another way would be to use `EXCLUDE_PATTERNS`. Might make things easier in the long run so any new files that are added in `control-plane` will automatically get picked up in the Doxygen output.